components/Repositories/RossChart.tsx

```diff
@@ -0,0 +1,137 @@
+import { FaUsers } from "react-icons/fa6";
+import { Bar, BarChart, CartesianGrid, ResponsiveContainer, Tooltip, TooltipProps, XAxis, YAxis } from "recharts";
+import { NameType, ValueType } from "recharts/types/component/DefaultTooltipContent";
+import { useMemo } from "react";
+import Card from "components/atoms/Card/card";
+import SkeletonWrapper from "components/atoms/SkeletonLoader/skeleton-wrapper";
+import { humanizeNumber } from "netlify/og-image-utils";
+
+type RossChartProps = {
+  stats: RepositoryRoss | undefined;
+  isLoading: boolean;
+  error: Error | undefined;
+  range: number;
+  rangedTotal: number;
+  className?: string;
+};
+
+export default function RossChart({ stats, rangedTotal, isLoading, error, range, className }: RossChartProps) {
+  const rangedAverage = useMemo(() => (rangedTotal / range).toPrecision(2), [rangedTotal, range]);
+
+  const weeklyData = useMemo(() => {
+    const result: { bucket: string; new: number; returning: number; internal: number }[] = [];
+    stats?.contributors.reverse().map((week) => {
```

Comment:
Use a `forEach` if we're not mapping back to another value directly

```suggestion
    stats?.contributors.reverse().forEach((week) => {
```

Reply:
Any reason the stats don't come in with the contributors already reversed? Or maybe there's a reason the API doesn't return the contributors in the proper order?

Reply:
This is now inline maps!